tests/test_wicks_wampo_tpb.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import BOARD, PASSED
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.wicks_wampo_tpb import WicksWampoTPBSpider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "wicks_wampo_tpb.html"),
+    url="https://www.wampo.org/transportation-policy-body",
+)
+spider = WicksWampoTPBSpider()
+
+freezer = freeze_time(datetime(2024, 3, 14, 14, 1))
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+parsed_item = parsed_items[0]
+freezer.stop()
+
+
+def test_title():
+    assert parsed_item["title"] == "WAMPO Transportation Policy Body Meeting"
+
+
+def test_description():
+    assert parsed_item["description"] == ""
+
+
+def test_start():
+    assert parsed_item["start"] == datetime(2024, 1, 9, 15, 0)
+
+
+def test_end():
+    assert parsed_item["end"] is None
+
+
+def test_time_notes():
+    assert parsed_item["time_notes"] == ""
+
+
+def test_id():
+    assert (
+        parsed_item["id"]
+        == "wicks_wampo_tpb/202401091500/x/wampo_transportation_policy_body_meeting"
+    )
+
+
+def test_status():
+    assert parsed_item["status"] == PASSED
+
+
+def test_location():
+    assert parsed_item["location"] == {
+        "name": "Wichita Area Metropolitan Planning Organization",
+        "address": "271 W 3rd St N, Wichita, KS 67202",
+    }
+
+
+def test_source():
+    assert parsed_item["source"] == "https://www.wampo.org/transportation-policy-body"
+
+
+def test_links():
+    assert parsed_item["links"] == [
+        {
+            "href": "https://www.wampo.org/_files/ugd/bbf89d_bc9c575ffcd9480ca7bd7b8c7611857b.pdf",  # noqa
+            "title": "Agenda",
+        },
+        {
+            "href": "https://www.wampo.org/_files/ugd/bbf89d_8cf9fc33872e44d2a7ade0d2db6e5fae.pdf",  # noqa
+            "title": "Min",
+        },
+        {"href": "https://youtu.be/LsMI1EClvnI", "title": "Re"},
+        {"href": "https://youtu.be/LsMI1EClvnI", "title": "cording"},
```

Comment:
These 2 links should be unified into a single link: 
```json
{"href": "https://youtu.be/LsMI1EClvnI", "title": "Recording"}
```

Reply:
@SimmonsRitchie I noticed that the page renders complex html structure for a single link, eg. for [Agenda](https://www.wampo.org/_files/ugd/bbf89d_bc9c575ffcd9480ca7bd7b8c7611857b.pdf) link
<img width="1684" alt="WAMPO_Agenda_link" src="https://github.com/City-Bureau/city-scrapers-wichita/assets/1421838/09eea09a-b322-4ba5-be92-0208a769d831">
And for [Recording](https://youtu.be/LsMI1EClvnI) link, it breaks into 2 `<a>` elements. So I think you should rework on the `_parse_links` function.